src/GovUk.Education.ExploreEducationStatistics.Data.Model/Location.cs

```diff
@@ -247,5 +247,95 @@ public PlanningArea PlanningArea
+                   && ParliamentaryConstituency_Name == other.ParliamentaryConstituency_Name 
+                   && Provider_Code == other.Provider_Code 
+                   && Provider_Name == other.Provider_Name 
+                   && Region_Code == other.Region_Code 
+                   && Region_Name == other.Region_Name 
+                   && RscRegion_Code == other.RscRegion_Code 
+                   && School_Code == other.School_Code 
+                   && School_Name == other.School_Name 
+                   && Sponsor_Code == other.Sponsor_Code 
+                   && Sponsor_Name == other.Sponsor_Name 
+                   && Ward_Code == other.Ward_Code 
+                   && Ward_Name == other.Ward_Name 
+                   && PlanningArea_Code == other.PlanningArea_Code 
+                   && PlanningArea_Name == other.PlanningArea_Name;
+        }
+
+        public override bool Equals(object? obj)
+        {
+            if (ReferenceEquals(null, obj)) return false;
+            if (ReferenceEquals(this, obj)) return true;
+            if (obj.GetType() != this.GetType()) return false;
+            return Equals((Location)obj);
+        }
+
+        public override int GetHashCode()
+        {
+            var hashCode = new HashCode();
+            hashCode.Add(Id);
+            hashCode.Add((int)GeographicLevel);
+            hashCode.Add(Country_Code);
+            hashCode.Add(Country_Name);
+            hashCode.Add(EnglishDevolvedArea_Code);
+            hashCode.Add(EnglishDevolvedArea_Name);
+            hashCode.Add(Institution_Code);
+            hashCode.Add(Institution_Name);
+            hashCode.Add(LocalAuthority_Code);
+            hashCode.Add(LocalAuthority_OldCode);
+            hashCode.Add(LocalAuthority_Name);
+            hashCode.Add(LocalAuthorityDistrict_Code);
+            hashCode.Add(LocalAuthorityDistrict_Name);
+            hashCode.Add(LocalEnterprisePartnership_Code);
+            hashCode.Add(LocalEnterprisePartnership_Name);
+            hashCode.Add(MayoralCombinedAuthority_Code);
+            hashCode.Add(MayoralCombinedAuthority_Name);
+            hashCode.Add(MultiAcademyTrust_Code);
+            hashCode.Add(MultiAcademyTrust_Name);
+            hashCode.Add(OpportunityArea_Code);
+            hashCode.Add(OpportunityArea_Name);
+            hashCode.Add(ParliamentaryConstituency_Code);
+            hashCode.Add(ParliamentaryConstituency_Name);
+            hashCode.Add(Provider_Code);
+            hashCode.Add(Provider_Name);
+            hashCode.Add(Region_Code);
+            hashCode.Add(Region_Name);
+            hashCode.Add(RscRegion_Code);
+            hashCode.Add(School_Code);
+            hashCode.Add(School_Name);
+            hashCode.Add(Sponsor_Code);
+            hashCode.Add(Sponsor_Name);
+            hashCode.Add(Ward_Code);
+            hashCode.Add(Ward_Name);
+            hashCode.Add(PlanningArea_Code);
```

Comment:
If we order these alphabetically both here and above it makes it a little easier to check the presence of them against a list of all the fields we expect.

Reply:
Moved Planning Area into the correct alphabetical place in Equals and GetHashCode methods, and in the order of fields themselves in Location.  Left GeographicLevel and Id where they were.